modules/example-apache-httpd-configuration.adoc

```diff
@@ -55,6 +57,35 @@ echo "Content-Type: application/json"
 echo ""
 echo '{"error": "Login failure"}'
 exit 0
+
+htpasswd -bc passwords basicauth-xxia1 redhat
+htpasswd -b passwords basicauth-xxia2 redhat
```

Comment:
> Direct link to doc preview: https://deploy-preview-

Hmm, when I review, I review PR "Files changed" and also review the `doc preview` you gave: https://deploy-preview-... . Do you self review the `doc preview` you gave? If you did, it would be obvious to find that the commands wrongly fall under `Example /var/www/cgi-bin/fail.cgi` within its closing dashes `---`. Frankly speaking, I several times suggest such not-hard issue be found in PR author's self review.

Reply:
Sorry I am replying so late. I was learning more about pods and gaining a better understanding. I tested the commands as you had presented them in the bug and received a CrashLoopBackOff error after running the oc get pod command. I have added my full list of commands to the bug as well if you would like to verify. I'm not sure if I may have missed something, but these are the results I get when inputting the commands: 
![CrashLoopBackOff](https://user-images.githubusercontent.com/96480979/165986485-d1c426d7-9ad3-44ae-8ec8-e5d0dd5d1fdb.png)

Also, I wanted to ask how did you set your environment? I had to use export to ensure the environment variables and functions passed, as shown below:
![Evironment Variables](https://user-images.githubusercontent.com/96480979/165987187-8ad1b8b8-6bd6-4976-8a72-f73962768fd7.png)

Reply:
Thank you!! I have updated the PR after successfully testing the pods. Please let me know if this looks good! :)